tools/__tasks__/compile/conf/clean.js

```diff
@@ -0,0 +1,9 @@
+const path = require('path');
+const rimraf = require('rimraf');
+
+const {root} = require('../../config').paths;
+
+module.exports = {
+    description: 'Clear template rendering artefacts',
+    task: () => rimraf.sync(path.resolve(root, 'common', 'conf', 'assets'))
```

Comment:
yeah i agree with this – once this part is complete, i'm going to look at exactly what happens in the tasks, it's exposing a few oddities like that